tests/integration/test_integration.sh

```diff
@@ -1,62 +1,218 @@
-#!/bin/sh
+#!/bin/bash
 
 # Exit on failure
 set -e
 
 # Bootstrap
-## Aliases
-loadModelDirectory="./tests/integration/models/load"
 mcd="pipenv run python -m mcd postgresql+psycopg2://postgres:travisci@localhost:5432/postgres"
+modelDirectory="./tests/integration/models"
+loadModelDirectory="$modelDirectory/load"
+transformModelDirectory="$modelDirectory/transform"
+
+# Generate a pseudo UUID
+# https://gist.github.com/markusfisch/6110640
+NewUUID () {
+    local N B C='89ab'
+
+    for (( N=0; N < 16; ++N ))
+    do
+        B=$(( $RANDOM%256 ))
+
+        case $N in
+            6)
+                printf '4%x' $(( B%16 ))
+                ;;
+            8)
+                printf '%c%x' ${C:$RANDOM%${#C}:1} $(( B%16 ))
+                ;;
+            3 | 5 | 7 | 9)
+                printf '%02x-' $B
+                ;;
+            *)
+                printf '%02x' $B
+                ;;
+        esac
+    done
+
+    echo
+}
+
+LogErrorAndExit () {
+    local errorText=$1
+    local errorCode=1
+
+    >&2 echo ${errorText}
+    exit $errorCode
+}
+
+AssertAreEqual () {
+    local subject=$1
+    local actual=$2
+    local expected=$3
+    if [ "$actual" == "$expected" ]
+    then
+        echo "PASS: "$subject" as expected"
+    else
+        LogErrorAndExit "ERROR: "$subject" was expected to be "$expected", actual was "$actual""
+    fi
+}
 
 InitExecution () {
-    executionId=$($mcd init-execution)
+    local executionId=$($mcd init-execution)
+
+    if [ ${#executionId} -ne 36 ]
```

Comment:
```suggestion
    if [ ${#executionId} != 36 ]
```

to keep convention with the rest of the file